src/coreclr/vm/vm.natvis

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+Licensed to the .NET Foundation under one or more agreements.
+The .NET Foundation licenses this file to you under the MIT license.
+-->
+
+
+<AutoVisualizer xmlns="http://schemas.microsoft.com/vstudio/debugger/natvis/2010">
+    <Type Name="HolderBase&lt;*&gt;">
+        <DisplayString>{*m_value}</DisplayString>
+        <Expand>
+            <ExpandedItem>m_value</ExpandedItem>
```

Comment:
do you need `ExpandedItem` for this? May be a screenshot will help. You can also consider adding `StringView` for better copy paste experience.

Reply:
Since `HolderBase` doesn't usually hold a string, I think `StringView` would be the wrong element to add here. I think `DisplayString` handles that case. I'll change it to `m_value`.

Reply:
Ok..And then what is the purpose of `ExpandedItem`? Isn't `DisplayString` sufficient?

Reply:
`ExpandedItem` basically removes a level of indirection in the expanded tree when looking in the locals/auto/watch view, so when the holder object is expanded, we get directly to the held value.